File: iCovieAdmin-Ng/New folder/src/app/core/auth/_models/Masters/propertycategory.model.ts


export class PropertyCategoryModel {
    id: number;    
    propertycategory:string;   
    cid:number; 
    isCore: boolean = false;    
    clear(): void {
        this.id= undefined;
        this.propertycategory= '';        
        this.cid = undefined;      
        this.isCore=false;
	}
}

File: iCovieApi/iCovieApi/Models/Master/PropertyCategoryModel.cs
using System;

using System;
using Newtonsoft.Json;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace iCovieApi.Models
{
    public class PropertyCategoryModel
    {
        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
        public int id { get; set; }

        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
        public string propertycategory { get; set; }

        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
        public string shortname { get; set; }


        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
        public int cid { get; set; }
    }
}